src/controllers/tasks.ts

```diff
@@ -0,0 +1,20 @@
+import { Response } from 'express';
+import { CustomError } from '../helper';
+import { getTasksByUserId } from '../database/query/tasks';
```

Comment:
import it from the index file 

```suggestion
import { getTasksByUserId } from '../database/query';
```